nmproxy/router/iptables_linux.go

```diff
@@ -25,11 +25,10 @@ const (
 )
 
 type iptablesManager struct {
-	ipv4Client   *iptables.IPTables
-	ipv6Client   *iptables.IPTables
-	ingRules     serverrulestable
-	defaultRules ruletable
```

Comment:
Did you want to remove this defaulRules too?

Reply:
I only deleted that to fix a staticcheck.